src/docs/self-hosted/external-storage.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: External Storage
+---
+
+<!-- Hello! If you're reading this, you're in luck because I can't decide whether to make.. wait let me copy the text from Discord.
+  
+  I got some time before Monday to write up some docs about setting up an S3 storage for selfhosted instance, but I can't decide whether I should put it under a big "External Services" page, in which people can include external postgres, external redis, and that kind of things; or should I put it under a page called "External Storage"?
+
+  There. Please help me decide this. I'll delete this comment afterwards -->
+
+<Alert title="Note" level="info">
+    After changing configuration files, re-run the <code>./install.sh</code> script, to rebuild and restart the containers. See the <Link to="/self-hosted/#configuration">configuration section</Link> for more information.
+</Alert>
+
+<!-- Should we add a description about what "external storage" is? -->

```

Comment:
I think we can safely assume that people can find that out for themselves if they are looking at this page